@W-22102616: Add note for ARM64 architectures in FIPS document (#3355)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/fips-140-3-compliance-support.adoc

bcpkix-fips-2.1.10.jar
----
+

> . Configure Security Properties in `java.security`

Reviewer: ```suggestion
. Configure the security properties in `java.security`.
```

+
** Security providers:
+
----
security.provider.1=org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
security.provider.2=org.bouncycastle.jsse.provider.BouncyCastleJsseProvider fips:BCFIPS
----
+
[NOTE]
--

> On ARM64 (AArch64) architectures, you must also register the `SUN` provider. BC-FJA 2.1.2 has no native library support for AArch64, and the JVM's `securerandom.strongAlgorithms` property references `NativePRNGBlocking:SUN` and `DRBG:SUN`. Without the `SUN` provider registered, the JVM can't resolve these algorithms and exits silently before Mule's logging initializes.

Reviewer 2: ```suggestion
On ARM64 (AArch64) architectures, you must also register the `SUN` provider. BC-FJA 2.1.2 does not include a `SecureRandom` implementation for AArch64 (its native cryptographic library targets x86_64 only). The  JVM's `securerandom.strongAlgorithms` property defaults to. NativePRNGBlocking:SUN` and `DRBG:SUN`, and without a provider that supplies these algorithms, the JVM fails during early bootstrap — before Mule's
  logging initializes.
```

Add `SUN` as security provider 3:

----
security.provider.1=org.bouncycastle.jcajce.provider.BouncyCastleFipsProvider
security.provider.2=org.bouncycastle.jsse.provider.BouncyCastleJsseProvider fips:BCFIPS
security.provider.3=SUN
----
--
+

> . Set key manager and trust manager algorithms:

Reviewer: ```suggestion
. Set the key manager and trust manager algorithms:
```
